package.json

```diff
@@ -34,7 +34,7 @@
     "async": "3.2.4",
     "flaverr": "^1.10.0",
     "machine": "^15.2.2",
-    "mongodb": "3.7.3",
+    "mongodb": "6.3.0",
```

Comment:
so.... tempted... to loosen this SVR (semantic version range) to a `^6.3.0`, but better to stick to the convention everywhere of only doing that for dependencies maintained by the Sails core team.  (for now.  only way I see of improving this is to add smarter package-lock documentation for folks, rather than staying out of it.  But I think that's not worth messing with now- it'll just make things harder for people instead of easier.  Better to have the core team continue to just stay on top of vulnerabilities for the moment.)

Reply:
Yeah I was tempted too @mikermcneil but decided to follow suit as we've done for `mongodb 3.7.3`